[IMP] subscriptions: update subscription plans

meng-odoo opened the pull request:
Update to the emergency subscriptions update for Odoo 16 (eliminating subscription templates from Odoo 15).
Related to task #[3222232](https://www.odoo.com/web#id=3222232&menu_id=4720&cids=3&action=333&active_id=3835&model=project.task&view_type=form)

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/4343).

hojo-odoo commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,96 @@
+
+The subscription plan form contains the following options:
+
+- The name of the plan: Enter a name for the subscription plan at the top of the page.
```
```suggestion
- Title: Enter a name for the subscription plan at the top of the page.
```

hojo-odoo commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,96 @@
+  quotation to never expire.
+- :guilabel:`Online Confirmation`: Check the boxes next to :guilabel:`Signature` or
+  :guilabel:`Payment` to enable the customer to confirm their subscription order by signing or
+  paying for the quotation. Enable both to leave the choice to the customer. Enable neither to only
```
```suggestion
  paying for the quotation. Enable both to leave the choice up to the customer. Enable neither to only
```

hojo-odoo commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,96 @@
+  available here are the same ones that are configured in :menuselection:`Subscriptions -->
+  Configuration --> Recurrence Periods`.
+
+Selecting a :guilabel:`Recurrence` turns the quotation template into a subscription plan and enables
```
```suggestion
Selecting a :guilabel:`Recurrence` period turns the quotation template into a subscription plan and enables
```

hojo-odoo commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,96 @@
+From the *Sales* dashboard, click :guilabel:`New` to create a new quotation. Then, select a
+subscription plan in the :guilabel:`Quotation Template` field.
+
+All subscription orders will appear on the *Subscriptions* dashboard regardless of whether they were
```
Same as above.

hojo-odoo reviewed (changes requested):
Hey @meng-odoo,

Great job on this! Most of my comments are pretty minor. Make sure to decapitalize all of the words that follow a colon.

Thanks,
hojo

meng-odoo commented:
Thanks for your review @hojo-odoo! FYI I followed all of the colons with a complete sentence so that they could properly be in sentence case (have a capitalized first letter).

Hi @samueljlieber, this is ready for technical review.

samueljlieber commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,97 @@
+
+  - If the duration is :guilabel:`Forever`, then the subscription plan will continually renew until
+    either the customer or the company manually ends the subscription.
+  - If the duration is :guilabel:`Fixed`, then enter an :guilabel:`End After` date, which
+    determines the amount of time after which the subscription will automatically end.
```
early 100th char line break

samueljlieber commented on content/applications/sales/subscriptions/plans.rst:
```diff
@@ -2,45 +2,97 @@
+- :guilabel:`Self Closable`: Check this box to enable the customer to terminate their subscription
+  from the :doc:`customer portal
+  </applications/websites/ecommerce/ecommerce_management/customer_accounts>`.
+- :guilabel:`Automatic Closing`: Enter the number of days after which *unpaid* subscriptions
+  *past* the due date are automatically closed.
```
early 100th char line break

samueljlieber reviewed (changes requested):
Hi @meng-odoo 👋 I made just a couple of technical suggestion on this PR, please review them below and let me know if you have any questions. 

Note on images: Since were improving this doc, can you please resize the images used in `plans.rst` to be a maximum of 933px wide, ideally 768px wide. 

Thank you! 🙏

meng-odoo commented:
Thank you @samueljlieber! I incorporated your edits (and added an image while I was at it) in the latest commit. Requesting your review again just in case :)

samueljlieber reviewed (approved):
Hi @meng-odoo! This PR is good to go 🚀 great job! @StraubCreative sending this to you for final review!
